webapp/src/components/surveyDateSelector/surveyDateSelector.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {format} from 'date-fns';
+
+import DatePicker from 'components/common/datePicker';
+
+import './style.scss';
+
+export type Props = {
+    value: Date
+    onChange?: (value: Date) => void
+};
+
+const SurveyDateSelector = ({value, onChange}: Props) => {
+    const onSelectHandler = useCallback((newValue: Date) => {
+        if (onChange) {
+            onChange(newValue);
+        }
```

Comment:
This can be simplified to `onChange?.(newValue);`

Reply:
```suggestion
        onChange?.(newValue);
```
Always good to make it easy to change ;)

Reply:
I'll go a step further actually. Couldn't we just pass `onChange` into the `DatePicker`'s `onSelect` prop? It doesn't look like we need to declare a new handler here

Reply:
Done @hmhealey 's suggestion. There was nothing else happening in the extra change handler to just passed prop change handler directly to the component. #